skfp/fingerprints/lingo.py

```diff
@@ -0,0 +1,179 @@
+import hashlib
+import re
+from collections import defaultdict
+from collections.abc import Sequence
+from numbers import Integral
+from typing import Optional, Union
+
+import numpy as np
+from rdkit.Chem import Mol
+from scipy.sparse import csr_array
+from sklearn.utils import Interval
+
+from skfp.fingerprints.base import FingerprintTransformer
+from skfp.validators import ensure_smiles
+
+
+class LingoFingerprint(FingerprintTransformer):
+    """
+    Lingo fingerprint.
+
+    The Lingo fingerprint [1]_ is a hashed fingerprint that checks the occurrences of
+    substrings of a given length in a SMILES string. Original paper uses raw dictionaries os output.
+    Our implementation returns constant-length hashed fingerprints. The hash function used in our
+    implementation is SHA-1.
```

Comment:
Do not use first person: `Original paper uses raw dictionaries as output, but here constant-length hashed fingerprints are returned`. Also, `os` is a typo.

Second sentence can he shorter: `SHA-1 hash function is used here`.